### DISCUSSION TITLE
Welcome to dj-shop-cart Discussions!

### DISCUSSION THREAD

**Discussion by Tobi-De** [Tobi-De/dj-shop-cart#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Undeadtaker** [comment]
Hey I just saw your implementation of the cart object and I'm impressed. I will definitely be using this in my project thank you very much for saving me a bunch of time!

**Reply by Tobi-De** [reply]
Thanks @Undeadtaker, glad you liked it.
